pkg/constants/constants.go

```diff
@@ -109,8 +109,8 @@ const (
 	SELinuxTargetedPolicyPath  = SELinuxTargetedPath + "/policy"
 
 	// Kernel and initrd paths are arbitrary and coupled to grub.cfg
-	ISOKernelPath    = "/boot/kernel"
-	ISOInitrdPath    = "/boot/initrd"
+	ISOKernelPath    = "/boot/x86_64/loader/linux"
```

Comment:
Can we get the actual architecture here instead of hard-coding `x86_64` ?!

(Yes, this is technically irrelevant as only x86_64 supports PXE booting in SUSE Manager, but it would make for cleaner code 😉 )

Reply:
Oh apologies, you mentioned this is only supported for `x86_64`, so we can use the `/boot/x86_64/loader` dir in `amd64` case, or revert to the previous `/boot` for all other cases

Reply:
No, I wouldn't revert for non-x86_64. Then we would have two code pathes.

Reply:
> I guess I could fetch runtime.GOARCH and map amd64 (if any) to x86_64, but then should I also map the other values? Would that make sense?

I see, probably too much effort. (I had hoped for a simple fix)

Let's merge as-is